Index.html

```diff
@@ -217,7 +220,8 @@ <h1 class="bottom-line">Skills</h1>
         <hr>
     </ul>
     </article>
-    <article class="header-img">
+    
+    <footer class="header-img">
         <h1 class="index-h1">Contact me</h1>
         <p>If you have an application you are interested in developing,
            a feature that you need built or a project that needs coding.
```

Comment:
- [ ] Kindly put an `action` and a `method` attribute on your form element so it can be integrated with the [Formspree](https://formspree.io/html/) service. You can get the form's contents in your email if the Formspree service is integrated successfully. 🤝️
**Example**
```
<form action="https://formspree.io/f/{form_id}" method="POST">
  ...
</form>
```

- [ ] Kindly use the correct `type` attribute on the email input to have the inbuilt HTML validations. `type="email"` instead of `type="text"`

- [ ] Kindly apply the necessary validations on the form inputs as stated in the project requirement. You can use the `maxlength` and `required` attributes to achieve such validations. 😎️

**Project requirement**
![image](https://user-images.githubusercontent.com/17243325/207247542-904fe0e5-2ffe-4387-acf9-48200ed149a4.png)

- [ ] Kindly use an `input` element with `type="submit"` attribute or a `button` element instead of an `a` element for your form button so the form can be submitted and collected by the Formspree service. 🤓️

- [ ] One of the objectives of this project is to understand how to parse a Figma design. Kindly try to stick to the template as much as possible. 

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/207249158-aae2c7ef-008a-4eb6-8726-0450505aa8b8.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/207249154-75339188-c6cb-4a8b-927e-f977445f9f98.png" width="300">|

- [ ] Kindly use the correct color and adjust the spacing you have on the left and right of the section.